src/python/pants/backend/scala/dependency_inference/scala_parser_test.py

```diff
@@ -48,6 +48,7 @@ def rule_runner() -> RuleRunner:
     return rule_runner
 
 
+@logging
 def test_parser_simple(rule_runner: RuleRunner) -> None:
```

Comment:
It might be good to start creating more methods in future changes... this one is no longer simple.

Reply:
Good point. And if it ever even was simple in the first place ...